BoxContentSDK/BoxContentSDKTests/BOXCollaborationCreateRequestTests.m

```diff
@@ -21,7 +21,8 @@ @implementation BOXCollaborationCreateRequestTests
 - (void)test_that_request_with_user_id_has_expected_URLRequest
```

Comment:
Both file and folder methods actually use the same initializers, just with a different string for itemType. I just reviewed the test cases and I think we have pretty good coverage there.